### PR TITLE
Added additional information to Invitations page

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information to Invitations page

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/invitations.html

### CODE REVIEW

File: src/marketing/invitations.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ When invitations are enabled, customers can send and view [invitations]({% link
 
 1. In the upper-right corner, click **Add Invitations**.
 
-1. On the next screen, enter email addresses to invite new customers, add a custom message, choose a sender, and select an invitee group.
+1. On the next screen, enter email addresses to invite new customers, add a custom message, choose a sender, and select an invitee group. If you have multiple store views, set the **Send From** chooser to establish from what store view an invitation will be sent.
```
Comment (leslietilling) [inline src/marketing/invitations.md:21]:
```suggestion
1. On the next screen, enter email addresses to invite new customers, add a custom message, choose a sender, and select an invitee group.

   If you have multiple store views, use the **Send From** option to specify the store view from which an invitation is sent.
```

File: src/marketing/invitations.md [hunk 2]
```diff
@@ -27,6 +27,26 @@ When invitations are enabled, customers can send and view [invitations]({% link
 
 In the [Private Sales Reports]({% link reports/private-sales-reports.md %}) section, you can see the number of invitations sent during a specified period, or customers to whom you have sent invitations.
 
+## Discard Invitations for single entity
```
Comment (leslietilling) [inline src/marketing/invitations.md:30]:
```suggestion
## Discard invitations for single entity
```

File: src/marketing/invitations.md [hunk 3]
```diff
@@ -27,6 +27,26 @@ When invitations are enabled, customers can send and view [invitations]({% link
...
+
+1. On the _Admin_ sidebar, go to **Marketing** > _Private Sales_ > **Invitations**.
+
+1. Find the needed invitation using filters and open it en edit mode.
+
+1. In the upper-right corner, click <span class="btn">Discard Invitation</span>.
+
+1. To confirm the action, click <span class="btn">OK</span>.
+
+## Discard Invitations for multiple entities
```
Comment (leslietilling) [inline src/marketing/invitations.md:40]:
```suggestion
## Discard invitations for multiple entities
```

File: src/marketing/invitations.md [hunk 4]
```diff
@@ -27,6 +27,26 @@ When invitations are enabled, customers can send and view [invitations]({% link
...
+
+1. On the _Admin_ sidebar, go to **Marketing** > _Private Sales_ > **Invitations**.
+
+1. Find the invitations to be discarded and select them.
```
Comment (leslietilling) [inline src/marketing/invitations.md:44]:
```suggestion
1. Find and select the invitations to be discarded.
```

File: src/marketing/invitations.md [hunk 5]
```diff
@@ -27,6 +27,26 @@ When invitations are enabled, customers can send and view [invitations]({% link
...
+
+1. Choose **Discard Selected** in the __Actions__ drop-down and click **Submit**.
```
Comment (leslietilling) [inline src/marketing/invitations.md:46]:
```suggestion
1. At the top-left, use the **Actions** menu to choose **Discard Selected** and click **Submit**.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
minor edits for style and clarity

**Comment by DmitryTsymbal** [comment]
Hello @leslietilling , thank you for the review. I have applied requested changes, please check.
Thanks!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**